### PR TITLE
T3T1 - some copy and flow updates

### PR DESCRIPTION
**Pull request by obrusvit**
This PR:
- makes sure that "Wallet Created Successfully" screen is shown even when `skip_backup` is used in the
- separates `prompt_backup` from `backup_intro` so that the explanatory screen can be shown according to Figma. This is an example for single-share backup: 
image
- forbids the user to `"Cancel backup"`
- adds some other smaller improvements to reset/recovery flows.

Some click tests not passing - in progress.

### CODE REVIEW

**Comment by Hannsek** [comment]
Maybe you can incorporate the first item into this pr from [this issue](https://github.com/trezor/trezor-firmware/issues/3880)?

File: core/embed/rust/src/ui/model_mercury/flow/prompt_backup.rs [hunk 1]
```diff
@@ -74,20 +74,16 @@ use crate::{
 };
 
 #[allow(clippy::not_unsafe_ptr_arg_deref)]
-pub extern "C" fn new_prompt_backup(n_args: usize, args: *const Obj, kwargs: *mut Map) -> Obj {
+pub extern "C" fn new_promp_backup(n_args: usize, args: *const Obj, kwargs: *mut Map) -> Obj {
```
[the remark this answers is not in the document]
Comment (obrusvit) [reply core/embed/rust/src/ui/model_mercury/flow/prompt_backup.rs]: Omission, reverted. 43b1e5c809ff1861420d92b07f74cefe868bfcae

File: core/embed/rust/src/ui/model_mercury/flow/confirm_reset_create.rs [hunk 2]
```diff
@@ -87,9 +87,11 @@ impl ConfirmResetCreate {
             .map(|msg| matches!(msg, FrameMsg::Button(_)).then_some(FlowMsg::Info))
             .one_button_request(ButtonRequestCode::ResetDevice.with_type("setup_device"));
 
+        // FIXME: TR::reset__cancel_create_wallet should be used but Button text on
+        // multiple lines not supported yet
```
[the remark this answers is not in the document]
Comment (obrusvit) [reply core/embed/rust/src/ui/model_mercury/flow/confirm_reset_create.rs:91]: Keeping the comment, but also wrote it to the issue.

**Comment by obrusvit** [comment]
I added `show_backup_intro` also to model T and TS3 a7340c4bd5d4d7ea0f529260c18beec3a30ada2b for
1. consistency of info delivered to the user across models
2. simplifying tests

**Comment by Hannsek** [comment]
This screen on the right do not suppose to be there anymore 
image

Also I see some weird changes: 
image

Another unwanted changes: 
image


However, I won't block this PR. Let's fix it in different one.

**Comment by obrusvit** [comment]
@Hannsek 

> This screen on the right do not suppose to be there anymore 

This screen will be shown only when you `trezorctl device setup`. In a flow initiated by Suite (skip backup first, then backup). It won't be visible

> Also I see some weird changes: 

Buttons in `active` state change color slightly. This is incomplete implementation of Figma design  - the background should also change color but it looked weird in some buttons so I kept it like this for now. However, even this small change is registered by the test recorder. I'm not sure about the random pixelation around though.

> Another unwanted changes:

These are according to Figma - Super-shamir not done though. In multishare, it looks nice.

**Comment by Hannsek** [comment]
> > Another unwanted changes:

> These are according to Figma - Super-shamir not done though. In multishare, it looks nice.

You mean this screen?

image

**Review by mmilata — APPROVED** [review]

**Pull request merged**